src/components/ProgressBar/ProgressBar.test.tsx

```
@@ -0,0 +1,503 @@
+      // Only percentage is shown in horizontal layout
+      const percentage = screen.getByText('60%');
+
+      expect(progressBar).toBeInTheDocument();
+      expect(percentage).toBeInTheDocument();
+    });
+  });
+
+  describe('Size variants', () => {
+    it('applies small size classes', () => {
+      const { container } = render(<ProgressBar size="small" value={50} />);
+      const progressBar = screen.getByRole('progressbar');
```

ESLint

[error] 'progressBar' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)